ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/spanning-tree.j2

```diff
@@ -2,6 +2,8 @@
 {% if spanning_tree is arista.avd.defined %}
 ## Spanning Tree Summary
 
+STP Root Super: **{{ spanning_tree.root_super | arista.avd.default(false) }}**
```

Comment:
Add an "arista.avd.defined" condition around this, so you don't change documentation unless the variable is set. Then you can skip the "arista.avd.default" and just print the value (true or false).